src/Microsoft.Azure.WebJobs.Host/Queues/StorageQueueMessageExtensions.cs

```diff
@@ -27,6 +27,10 @@ public static string TryGetAsString(this IStorageQueueMessage message)
                 // when the message type is binary, AsString will throw if the bytes aren't valid UTF-8.
                 value = null;
             }
+            catch (FormatException)
```

Comment:
Yeah, we can't do that yet until we move the SDK and Dashboard projects from TeamCity to AppVeyor. If somebody magically made that happen while I'm away on vacation, I'd buy them a case of beer :)